### DISCUSSION TITLE
change theme and need good  templates for cp

### DISCUSSION THREAD

**Discussion by rajputrv** [cpeditor/cpeditor#1127] [Development]
hi, I need a good template for cp and  want to get a cool theme

anyone who has good template to share please help and also recommend a good theme and how to use it

**Comment by md-shamim-ahmad** [comment]
Hi,
You can use CP Editor ```Light Fusion``` theme. It looks really awesome and I also use this theme.
Follow these rules to use it,
* First Press ```ctrl+p``` or click ```Options``` then go to ```Preferences```.
* Then go to ```Appearance```.
* Then go to ```General```.
* Then change UI Style  ```Light Fusion``` and also change Opacity ```full```.

I have a simple template code for c++
I am sharing the template code link here.
If you want, you can use it for CP.
[code link](https://pastebin.com/jN5f0LY1) or 
[code link](http://pastie.org/p/3Foa57GOUo886qkLJeZpRo)

**Reply by funcdfs** [reply]
theme template, not theme and template
